docs/source/plugins/development/publishing.md

```diff
@@ -0,0 +1,28 @@
+# Publishing a Plugin
+
+:::{index} Publishing a Plugin
+:::
+
+## Packaging Tutorial
+The best way to make your plugin available for others is by publishing it to a package manager registry.
+
+Use the [Python Packaging User Guide](https://packaging.python.org/en/latest/) to learn how to package and publish a package to PyPI.
+Specifically, the [Packaging Python Projects](https://packaging.python.org/en/latest/tutorials/packaging-projects/) details this process.
+
+## Configuring Entry Point
+Publishing the package will make it installable by package managers, 
+but an extra step is necessary to make the package discoverable as a plugin by Arelle.
+
+Arelle uses [`setuptools`](https://pypi.org/project/setuptools/) to discover packages that define entry points for Arelle.
+Entry points can be defined in `pyproject.toml` as described [here](https://setuptools.pypa.io/en/latest/userguide/entry_point.html#entry-points-for-plugins).
+For `ixbrl-viewer`, that looks like [this](https://github.com/Arelle/ixbrl-viewer/blob/master/pyproject.toml#L41-L43C1):
```

Comment:
Use a permalink URL. The referenced lines will end up out of sync at some point.